pad_type: ps3
num_of_buttons: 15 
axis_linear_x: 1
axis_linear_y: 0
axis_angular: 2
axis_linear_z: 3
scale_linear: 1.7
scale_angular: 3.1416
scale_linear_z: 20.0 
button_speed_up: 12
button_speed_down: 14
button_dead_man: 11
button_output_1: 15
button_output_2: 13
output_1: 1 
output_2: 2
button_ptz_tilt_up: 4
button_ptz_tilt_down: 6
button_ptz_pan_right: 5
button_ptz_pan_left: 7
button_home: 8
button_raise_elevator: 4
button_lower_elevator: 6
button_stop_elevator: 16
tilt_increment: 5
pan_increment: 5
button_kinematic_mode: 9
cmd_service_home:  /rb2_controller/home
check_message_timeout: true
